win32ss/user/ntuser/class.c

```diff
@@ -43,19 +43,19 @@ REGISTER_SYSCLASS DefaultServerClasses[] =
     FNID_DESKTOP,
     ICLS_DESKTOP
   },
-  { ((PWSTR)((ULONG_PTR)(WORD)(0x8003))),
+  { ((PWSTR)WC_SWITCH),
     CS_VREDRAW|CS_HREDRAW|CS_SAVEBITS,
     NULL, // Use User32 procs
-    sizeof(LONG),
+    sizeof(LONG_PTR), // See user32_apitest GetClassInfo, 0: Pointer to ALTTABINFO
```

Comment:
So it looks like it is this change that is sensitive to x86 vs. x64 ... (regarding the tests)